pkg/events/events.go

```diff
@@ -58,13 +100,37 @@ type EventType string
 var ErrMissingEventType = errors.New("unmarshalling error: missing event type")
 
 type jsonEvent struct {
+	EventVersion     Version          `json:"version"`
 	EventType        string           `json:"type"`
 	EventIssuedAt    time.Time        `json:"issuedAt"`
```

Comment:
Does this mean, the ref is part of the hash as well......????

Reply:
Nope, is removed before hashing. I'll add a documentation section explaining what should be included in the hash.